indico/modules/events/management/controllers/protection.py

```diff
@@ -63,42 +70,86 @@ def _process(self):
                                 endpoint='event_management.acl')
 
 
-class RHEventProtection(RHManageEventBase):
+class RHEventProtectionBase(RHManageEventBase):
+    def __init__(self):
+        super(RHEventProtectionBase, self).__init__()
+        selectable_permissions = {k: v for k, v in get_available_permissions(Event).viewitems() if v.user_selectable}
+        self.whitelisted_permissions = set(selectable_permissions.keys()) | {'edit', 'access'}
+        self.permissions_tree = {
+            'edit': {
```

Comment:
If this shares scope with actual permissions *anywhere*, please rename it to something like `_edit` or `_full_access` so it doesn't break if someone adds e.g. an 'edit' permission through a plugin.

Reply:
Same for the 'access' permission by the way (`_read_access`)